notebooks/3_SinglePlotDiagnostics/1_Ins_SingleModelVisualDiagnostics.ipynb

```diff
@@ -581,7 +581,7 @@
    "source": [
```

Comment:
<p>the sum should start from `i=0`, and the elements on the bottom should be squared.</p>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/bayesian-model-evaluation/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
Got more review than I was bargaining for :D Will fix tonight. Thank you